site/pages/migration-guide.mdx

```diff
@@ -32,6 +32,19 @@ We still support all Signers in the SDK!
 use the guide above to integrate with any signer of your choice.
 :::
 
+### Alchemy Transport
+
+A new `Transport` type has been added: `AlchemyTransport`. This impacts how Alchemy clients, middleware, and configs are created. The `AlchemyTransport` is a type of `viem` transport that makes it easier to configure your communication with Alchemy RPC.
+It supports splitting traffic between Alchemy's AA infra and other Node providers (if needed). The `AlchemyTransport` has been added to `@account-kit/infra` and is exported as `alchemy`.
+
+For Smart Account Clients, the impact of this change is that every `create*AlchemyClient` method has been updated to take in a `transport` property instead of `rpcUrl` or `apiKey` directly.
+
+The `alchemyFeeEstimator` and `alchemyUserOperationSimulator` middleware methods no longer take in a `ClientWithAlchemyMethods` (returned by `createAlchemyPublicRpcClient`) and instead take in an `AlchemyTransport`.
+
+Creating a config to be used with `@account-kit/core` or `@account-kit/react` has also been updated to accept an `AlchemyTransport` and the parameters of `createConfig` have been simplified as a result. Replace your `rpcUrl` or `apiKey` params with `transport: alchemy({...})`.
+
+For more detailed examples, see the relevant guides, depending on which package you are using.
```

Comment:
Should we be linking in "relevant guides"

Reply:
I'm gonna change this copy to `see relevant Quick Starts, depending... If you don't know where to get started see React` and link to react